### DISCUSSION TITLE
Fonts/ type faces

### DISCUSSION THREAD

**Discussion by mbierman** [isontheline/pro.webssh.net#342] [Q&A]
I know that I will feel stupid for asking once I know the answers, but how to I choose the type face and are they global or specific to each connection?

**Comment by isontheline** [comment]
No stupid question 😇

Yet fonts can't be chosen 😢

I've just released the support of [Powerlevel10k](https://github.com/romkatv/powerlevel10k) inside WebSSH 15.0.5.

Do you need another font?

**Reply by mbierman** [reply]
"Need" is a strong word. I saw that in the release notes and just assumed I could choose the font. What you selected isn't a bad choice.

**Reply by isontheline** [reply]
You're right that one of these issues was to be able to choose a font by the user.

I will reopen this issue for a future milestone.

If you use custom fonts in your daily work, you could post their names here.
